dimagi/pages/templates/svg/partner/oikoi grey with presentation attributes.html

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
can you eliminate some of the extra tags and comments in here please? see the other SVG files for reference. Also please close all the tags e.g. `<circle></circle>` remove any `<g>` tags and make sure the `svg` tag contains only the same attributes as the other examples?